doc/Changes.md

```diff
@@ -0,0 +1,6344 @@
+# GAP - Changes from Earlier Versions
+
+Release 4.11.0, ??-???-2020
+
+**The GAP Group**  
+Email: <support@gap-system.org>  
+Homepage: <https://www.gap-system.org>
+
+### Copyright
+
+Copyright © (1987-2020) for the core part of the **GAP** system by the
+**GAP** Group.
+
+Most parts of this distribution, including the core part of the **GAP**
+system are distributed under the terms of the GNU General Public
+License, see <http://www.gnu.org/licenses/gpl.html> or the file `GPL` in
+the `etc` directory of the **GAP** installation.
+
+More detailed information about copyright and licenses of parts of this
+distribution can be found in Section Reference: Copyright and License of
+the **GAP** reference manual.
+
+**GAP** is developed over a long time and has many authors and
+contributors. More detailed information can be found in Section
+Reference: Authors and Maintainers of the **GAP** reference manual.
+
+
+### Preface
+
+This document describes most essential changes from previous **GAP** releases.
+
+In addition to this manual, there is the **GAP** Tutorial (see
+[Tutorial: Preface](../../doc/tut/chap1.html#X874E1D45845007FE)) and the
+**GAP** Reference Manual (see Reference: Preface) containing detailed
+documentation of the mathematical functionality of **GAP**.
+
+A lot of the functionality of the system and a number of contributed
+extensions are provided as **GAP** packages, and each of these has its
+own manual. New versions of packages are released independently of
+**GAP** releases, and changes between package versions may be described
+in their documentation.
+
+
+### 1 Changes between **GAP** 4.10 and **GAP** 4.11
```

Comment:
I would suggest that we drop those chapter numbers. Esp. if we add new stuff at the top: then we'd have to renumber *all* sections with each release. (Removing them can be done via a sed or perl regex, I can also take care of it after merging)

I'd also make the exact version clear here, which is 4.11.0. In fact, how about just this:
```suggestion
### GAP 4.11.0
```
or this:
```suggestion
### Changes in GAP 4.11.0
```